packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -842,7 +855,10 @@ class ThemeData extends Diagnosticable {
         chipTheme,
         platform,
         materialTapTargetSize,
-        cursorColor
+        hashValues( // Too many values.
```

Comment:
Do you have to call `hashValues` again or can you just add tabBarTheme to the list?

Reply:
`hashValues` takes a max of 20 items, so I can't add `tabBarTheme` to the list of params, but instead I replaced `cursorColor` (the 20th param) with a new `hashValues(` call, which is the hash of both the `cursorColor` and the `tabBarTheme`.

(This is the same pattern used above).

Reply:
Word